views/graph/show.handlebars

```diff
@@ -1,110 +1,407 @@
+                    fadingIn = false
+                })
+                return true
+            }
+
+            $body.on('mousemove touchend', function (e) {
+                if (inVR()) return true
+                headerFadein()
+                if (!overHeader)
+                    queueFadeoutTimer()
+                return true
+            })
+
+            $header
+                .on('mouseenter', function () {
+                    clearFadeoutTimer()
+                    overHeader = true
+                    headerFadein()
+                })
+                .on('mouseleave', function () {
+                    overHeader = false
+                })
+
+            $body
+                .on('VRManInstructionsShown', function () {
+                    $header.hide()
+                    fadingIn = fadingOut = false
+                })
+                .on('VRManInstructionsHidden', function () {
+                    fadingIn = fadingOut = false
+                    if (siteUI.isPortraitLike()) {
+                        headerFadein()
+                        queueFadeoutTimer()
+                    }
+                })
+
+            setTimeout(function () {
+                if (VizorUI.isMobile.any() && !siteUI.isPortraitLike())
+                    headerFadeout()
+                else
+                    queueFadeoutTimer()
+            }, 300)
+        }
+
+        function bindVRBackButton() {
+            $('#backfromvr').off('.vizor').on('click.vizor', function (e) {
+                E2.core.webVRManager.onBackClick_()
+                siteUI.tagBodyClass()
+                return false
+            })
+        }
+
+        function bindShareButton() {
+            var $share = $('button#sharebutton')
+            if ($share.length < 1) return
+
+            $share
+                .on('click', function (e) {
+                    e.preventDefault()
+                    var html = E2.views.partials.embedVRplayer(playerUI.data)
+                    suspendVRcamera()
+                    var modal = VizorUI.modalOpen(html, "Share this", 'player_share doselect_all', enableVRcamera)
+                    modal
+                        .find('textarea, input')
+                        .on('mouseup touchup', function (e) {
+                            e.currentTarget.select()
+                            e.currentTarget.setSelectionRange(0, 9999)
+                            e.preventDefault()
+                            return true
+                        })
+                        .on('focus', function (e) {
+                            e.preventDefault();
+                            e.stopPropagation();
+                            return false
+                        })
+
+                    return false;
+                })
+        }
+
+        function amendVRManagerInstructions() {
+            if (!(E2 && E2.core && E2.core.webVRManager && E2.core.webVRManager.rotateInstructions)) {
```

Comment:
these things need to be in place for anything to work. don't check for presence of prerequisites :)